release/release_workflow.wdl

```diff
@@ -124,25 +124,19 @@ task versionPrep {
     }
 }
 
-task makeGithubRelease {
+task draftGithubRelease {
     input {
         String githubToken
         String organization
-        File cromwellJar
-        File womtoolJar
         String oldVersion
         String newVersion
     }
-
-    String cromwellJarName = basename(cromwellJar)
-    String womtoolJarName = basename(womtoolJar)
-
     command <<<
         set -e
         set -x
 
         # download changelog from master
-        curl https://raw.githubusercontent.com/~{organization}/cromwell/master/CHANGELOG.md -o CHANGELOG.md
+        curl --fail https://raw.githubusercontent.com/~{organization}/cromwell/master/CHANGELOG.md -o CHANGELOG.md
```

Comment:
[ToL] It looks like `--fail` has the side effect of suppressing error output - could make things harder to debug (last time, we figured out the problem from the output).

But `--fail` >> nothing so 🤷‍♂️